src/biogeophys/IrrigationMod.F90

```diff
@@ -478,6 +481,8 @@ subroutine translate_irrig_method_default
          irrig_method_default_int = irrig_method_drip
       case ('sprinkler')
          irrig_method_default_int = irrig_method_sprinkler
+      case ('flood')
+         irrig_method_default_int = irrig_method_flood													  
```

Comment:
I know you didn't invent `irrig_method_default_int`, but it doesn't appear to be used for anything.